### PR TITLE
Ambitions from Skyrat

### PR DESCRIPTION
**Pull request by keronshb**
## About The Pull Request

It's this
https://github.com/Citadel-Station-13/Citadel-Station-13/pull/14369

Please, it's been a fucking year just speedmerge this

## Why It's Good For The Game
It was asked, original reasons in the post

## Changelog
:cl:
/:cl:

### CODE REVIEW

File: code/controllers/configuration/entries/general.dm [hunk 1]
```diff
@@ -349,3 +349,8 @@
 /datum/config_entry/flag/dynamic_config_enabled
 
 /datum/config_entry/flag/station_name_needs_approval
+
+//ambition end
+	config_entry_value = 5
+/datum/config_entry/number/max_ambitions	// Maximum number of ambitions a mind can store.
```
Comment (Linzolle) [inline code/controllers/configuration/entries/general.dm]: these two lines are flipped around backwards for some reason

Comment (keronshb) [reply code/controllers/configuration/entries/general.dm]: ...weird.  Fixed.

**Comment by Linzolle** [comment]
Can confirm this works.

**Comment by jjpark-kb** [comment]
Just prefacing that I mainly play and contribute on Skyrat.

Ambitions was a decent system for antags, so that there were motivations behind what they were doing; however, there might be some performance issues and some discontent antags. Many antags we found dislikeed Ambitions because "RNG chose me to write an essay this shift," so you may find that antags start later than when they usually did.

Currently on Skyrat, we are using a system called OPFOR (Opposing Forces). It is similar to Ambitions, except it is applicable to everyone where people can request to do gimmicks.

If you go with Ambitions, I hope you enjoy it!

**Comment by keronshb** [comment]
> Just prefacing that I mainly play and contribute on Skyrat.
> 
> Ambitions was a decent system for antags, so that there were motivations behind what they were doing; however, there might be some performance issues and some discontent antags. Many antags we found dislikeed Ambitions because "RNG chose me to write an essay this shift," so you may find that antags start later than when they usually did.
> 
> Currently on Skyrat, we are using a system called OPFOR (Opposing Forces). It is similar to Ambitions, except it is applicable to everyone where people can request to do gimmicks.
> 
> If you go with Ambitions, I hope you enjoy it!

Thanks for the heads up!  I'm not too familiar with either system, but I remember people telling me that ambitions went through a few different changes.  I think this might be one of the earlier versions, which I'm not sure what to compare it to.  I definitely want to try out ambitions here to see if they work, and if not so well then maybe I'll check out OPFOR.

**Comment by Linzolle** [comment]
so the only weird thing i've noticed with this actually is the adding and removal of the Objectives and Ambitions verb; it seems to not show up in the IC panel sometimes, and also doesn't get removed when deleting the antag datum through the traitor panel.
Granted, for some reason when you aren't an antag and still have the objectives verb you can only add ambitions.

**Pull request merged**